### PR TITLE
attempted a restyle on the chat interface + mic button

### PR DESCRIPTION
**Pull request by almakantara**
this is a first attempt for test as main purpouse

### CODE REVIEW

**Comment by gcampax** [comment]
I like the new my skills page, especially with the trash button. We need to make sure it's accessible though.
I'm not sure what the config thing would point to? We don't have such a page yet, and for most skills there is nothing to configure.
I think the rounded corners are a bit too strong, especially in the grid view. Also I thought rounded corners were going out of fashion?
Small bug: the "Configure new skill" button is misaligned.
Finally, I saw you changed the general style of .panel, which is a [Bootstrap 3 panel component](https://getbootstrap.com/docs/3.4/components/#panels). We need to test in almond-cloud, which uses panels in more places (at the bottom of the front page for the suggested commands, in [Thingpedia](https://dev.almond.stanford.edu/thingpedia), maybe some other places I don't remember) to make sure everything looks good.

File: views/devices_list.pug [hunk 1]
```diff
@@ -32,8 +33,9 @@ block content
         div.clearfix.visible-md
       div.col-lg-4.col-md-6
         div.panel.panel-default.installed-dev
-          a(href='https://thingpedia.stanford.edu/thingpedia/devices/by-id/' + dev.kind).panel-heading
-            h2.panel-title= dev.name
+          div#cont_head
```
Comment (gcampax) [inline views/devices_list.pug:36]: Repeated ID?

File: views/error.pug [hunk 2]
```diff
@@ -4,6 +4,13 @@ block page_name
   | Error
 
 block content
-  p Sorry that did not work
-  p= message
-  a(href='javascript:history.back()').btn.btn-primary Go back
+  div.sp_blk
```
Comment (gcampax) [inline views/error.pug:7]: These class names are not very readable, and also don't follow the existing snake-case convention. I would expand them to full words.

File: views/layout.pug [hunk 3]
```diff
@@ -23,13 +23,16 @@ html
...
+      
       div.navbar-header
         button(class="navbar-toggle collapsed", data-toggle="collapse", data-target="#page-nav")
           span.sr-only Toggle navigation
           span.icon-bar
           span.icon-bar
           span.icon-bar
-        span.navbar-brand.scroller.hidden-xs.hidden-sm Almond
+        span.navbar-brand.scroller.hidden-xs.hidden-sm 
+          img.logo_top(src=Config.BASE_URL + '/images/almond.png')
```
Comment (gcampax) [inline views/layout.pug:34]: There is something wrong with the logo when I tried in Firefox, it overflows the navbar.

File: public/javascripts/conversation.js [hunk 4]
```diff
@@ -374,6 +375,7 @@ $(() => {
     function appendUserMessage(text) {
         container.append($('<span>').addClass('message message-text from-user')
             .text(text));
+        container.append('<div id="almond_thinking_container" ><div id="almond_thinking_txt"><p>Almond is thinking</p></div><div id="almond_thinking"><span></span><span></span><span></span><span></span><span></span><span></span><span></span></div></div>');
```
Comment (gcampax) [inline public/javascripts/conversation.js:378]: This doesn't work if you refresh the page: existing messages are resent by the server, and the spinner gets stuck.
Also, hardcoding the message here is problematic for a number of reasons, including translation.
I would do away with the message and entirely and only show the spinner, and also make it fewer dots. 3-5 should be enough.

**Review by gcampax — CHANGES_REQUESTED** [review]

File: views/devices_list.pug (hunk 1 above)
Comment (almakantara) [reply views/devices_list.pug:36]: that has to be a class, fixed

File: views/error.pug (hunk 2 above)
Comment (almakantara) [reply views/error.pug:7]: ok

File: views/layout.pug (hunk 3 above)
Comment (gcampax) [reply views/layout.pug:34]: > do you mean usability accessibility?

Yeah I mean accessibility (a11y) - as in, accessible to people with disabilities.

> design choice, it is not the same "content", it lead to a different path than the skills box

It looks off to me. Maybe it's just me.

File: public/javascripts/conversation.js (hunk 4 above)
Comment (almakantara) [reply public/javascripts/conversation.js:378]: > This doesn't work if you refresh the page: existing messages are resent by the server, and the spinner gets stuck.

Noticed later, trying to figure out a fix for that

> Also, hardcoding the message here is problematic for a number of reasons, including translation.

I know, that is a quick and dirty test, alredy implemented correctly

> I would do away with the message and entirely and only show the spinner, and also make it fewer dots. 3-5 should be enough.

ok

**Comment by gcampax** [comment]
Let's merge this and let's continue with the almond-cloud UI.

**Pull request merged**